README.md

```diff
@@ -1 +1,140 @@
+
+  <h3><b>Music Concert Project</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [music-concert] <a name="about-project"></a>
+
+
+
+**[music-concert]** This project is about The Night of Recovery Concert. A celebration of strength, resilience, and the power of music in healing. The project includes featured artist, important dates and other information. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+  <li>Responsive UI</li>
+    <li>html</li>
+    <li>css</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Zero linters error]**
+- **[Responsive mobile]**
+- **[Set up and design header section]**
+- **[Set up and design headline section]**
+- **[list of featured artist]**
+- **[Unique selling points for the event]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+ install a text editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Run this commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:folder/music-concert.git
+```
+
+
+### Deployment
+
+You can access this project using the link below:
+  
+  <a href="https://sulaimonaa.github.io/music-concert/">Check live demo</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/sulaimonaa)
+- Twitter: [@twitterhandle](https://twitter.com/aayodej007)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/sulaimonaa)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[About Night of recovery]**
+- [ ] **[Selling points of the event]**
+- [ ] **[Featured artist]**
+- [ ] **[event partners]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please give your rating or suggest to me how to make it better
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse team, coding partners and the code reviewers
+
```

Comment:
- [ ] You did great giving credit to Microverse. :+1: :+1: But being that the design was provided by `Cindy Shin ` and also pointed out In the project requirement, You are meant to give credit to the original creator of this design. 
> Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).

The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.